pilot/pkg/model/context.go

```diff
@@ -381,8 +381,8 @@ func DefaultProxyConfig() meshconfig.ProxyConfig {
 		ConfigPath:             ConfigPathDir,
 		BinaryPath:             BinaryPathFilename,
 		ServiceCluster:         ServiceClusterName,
-		DrainDuration:          types.DurationProto(2 * time.Second),
-		ParentShutdownDuration: types.DurationProto(3 * time.Second),
+		DrainDuration:          types.DurationProto(45 * time.Second),
+		ParentShutdownDuration: types.DurationProto(60 * time.Second),
```

Comment:
undo this please..we set this to 2/3 for a reason, after lots of experimentation

Reply:
this is the default set in our helm/istio/templates/configmap.yaml